### ISSUE TITLE
test

### ISSUE THREAD

**Issue by fernandolimati** [ColeMurray/medium-facenet-tutorial#6]
i do the example, but i cant test. Any help?

**Comment by fernandolimati** [comment]
i need an example to test 1 image on this trained classifier. HELP

**Comment by ColeMurray** [comment]
Hello Fernando,

Can you explain more in-depth where you are having difficulty testing images? Please provide the following information:

- Version of python 

- Structure of test data set

- Command used to execute test


With the code provided in, https://goo.gl/XRiaL2, you can modify the method "load_images_and_labels" to support your use case.

**Comment by fernandolimati** [comment]
I follow all steps on this site ( https://hackernoon.com/building-a-facial-recognition-pipeline-with-deep-learning-in-tensorflow-66e7645015b8 ). All OK.

After, i want to test only one image, to test with the classifier is OK. 
Im a student in python and deep learning, sorry about that.
Thanks about every think.

**Comment by tushalien** [comment]
@fernandolimati @ColeMurray  I am also having problems while testing a single image. All the functions seem to take a directory as input only.

**Comment by ColeMurray** [comment]
Hello Fernandolimati,

Can you please provide a stacktrace/logging of your issue and the command line arguments being used?

The tutorial is not intended to support single image evaluation, rather support training and evaluating a dataset. The test set is used to provide a metric about how well the model is performing on images out of sample.

You can experiment with the parameters
```--min-num-images-per-class 1```
and
```--split-ratio 1.0```

**Comment by ColeMurray** [comment]
@tushalien, the code assumes the data being used for test follows a similar structure.
Directory Structure
├── Tyra_Banks
│ ├── Tyra_Banks_0001.jpg
│ └── Tyra_Banks_0002.jpg
├── Tyron_Garner
│ ├── Tyron_Garner_0001.jpg
│ └── Tyron_Garner_0002.jpg
